src/tools/kdb/import.cpp

```diff
@@ -63,16 +63,9 @@ int ImportCommand::execute(Cmdline const& cl)
 
 	KeySet base;
 
-	// TODO: this should not be neccessary, but
-	// applying threeway strategies to a twoway merge is hard
-	base = KeySet();
+	base = existingKeys;
 
-	// TODO: for now we have to position this strategy manually
-	// to avoid meta information loss
-	MetaMergeStrategy metaStrategy(merger);
-	merger.addConflictStrategy(&metaStrategy);
-
-	helper.parseStrategies (cl, merger);
+	helper.configureMerger (cl, merger);
```

Comment:
> I am not aware of any tests that check if "kdb import" merges correctly.

check_import.sh should do this

> You told me about the differences of two-way merging which made it hard to implement "kdb import" ;)

Yes, but the differences should lie in the used strategies. For the two way stuff I had to implement a few strange strategies rarely used in any other scenario. But these strange strategies should be bundled in the import specific configurations (i.e. import and cut)